repos/system_upgrade/el7toel8/actors/firewalldreadlockdownwhitelist/actor.py

```diff
@@ -0,0 +1,26 @@
+from leapp.actors import Actor
+from leapp.models import FirewalldLockdownWhitelist
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
```

Comment:
you forgot to change import from Application to FactsPhaseTag

Reply:
Oops - thanks for catching. Pushed again.